### ISSUE TITLE
Edge thinks the Windows Installer might be unsafe

### ISSUE THREAD

**Issue by mundodisco8** [WerWolv/ImHex-Patterns#93]
Hi!

I just downloaded the Windows Installer with MS Edge, and the file triggered MS's SmartScreen protection.

The file can be reported as safe, but there's an option to certify as the author of the file that it is safe indeed. Maybe you would like to check that. Click on report as safe and follow the instructions in the form

![image](https://user-images.githubusercontent.com/8402707/222111661-e5a2a913-c5d4-45c3-8a2a-c24852c5d350.png)

Downloading with other browsers doesn't trigger the issue (e.g. Chrome)

**Comment by WerWolv** [comment]
Hey
I tried to do this countless times but it needs to be re-done every single time I make a new release to ImHex. The only way to truly make it not do that anymore is by paying the cert authorities tons of money and then sign the executable.
Since I don't want to invest that time and money into something stupid like this, I just stopped caring. The target audience of ImHex is smart enough to ignore these kind of warnings.
